Parsers/ASimAuthentication/Parsers/ASimAuthenticationGithub.yaml

```diff
@@ -0,0 +1,57 @@
+Parser:
```

Comment:
Please add entry for this parser in ASIM and IM union parsers:
https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/ASimAuthentication/Parsers/ASimAuthentication.yaml
https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/ASimAuthentication/Parsers/imAuthentication.yaml